Alignment/OfflineValidation/plugins/EopTreeWriter.cc

```diff
@@ -137,7 +137,7 @@ EopTreeWriter::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
    // get geometry
    edm::ESHandle<CaloGeometry> geometry;
    iSetup.get<CaloGeometryRecord>().get(geometry);
-   CaloGeometry* geo = (CaloGeometry*)(geometry.product());
+   CaloGeometry* geo = const_cast<CaloGeometry*>(geometry.product());
```

Comment:
are these const_casts really necessary?
Typically this is not allowed.

Reply:
I agree. Such a const_cast should not be done.